admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -1771,19 +1789,29 @@ var priceCalculation = (function() {
      * @param {jQuery} obj
      */
     impactTaxInclude: function (obj) {
-      var price = Tools.parseFloatFromString(obj.val());
-      var targetInput = obj.closest('div[id^="combination_form_"]').find('input.attribute_priceTI');
-      var newPrice = 0;
+      var priceTEInput = obj.closest('div[id^="combination_form_"]').find('input.attribute_priceTE');
+      var impactPriceTE = Number(Tools.parseFloatFromString(priceTEInput.val()));
 
-      if (!isNaN(price)) {
+      this.updateImpactTI(impactPriceTE, obj);
+    },
+
+    /**
+     * Computes the impact price tax included and update the related input
+     * @param {Number} impactPriceTE
+     * @param {jQuery} obj
+     */
+    updateImpactTI: function(impactPriceTE, obj) {
+      var impactPriceTI = 0;
```

Comment:
```suggestion
      var impactPriceTI = 0;
      var displayPrecision = 6;
```

Reply:
Actually I now use the scope variable `displayPricePrecision` which is fetched from data attribute Much better